csrc/device_lower/pass/index.cpp

```diff
@@ -1578,6 +1578,7 @@ void IndexLowering::handle(const MmaOp* mma) {
   constexpr int64_t core_matrix_outer_size = 8;
   Val* a = nullptr;
   Val* b = nullptr;
+  auto layout = lower_utils::getMmaLayout(mma);
```

Comment:
This might be more clear and removes one additional variable named `layout` :-D
```suggestion
  const auto& [unitdim_a, unitdim_b] = lower_utils::getMmaLayout(mma);
```

Reply:
fixed